packages/rum/src/index.d.ts

```diff
@@ -192,6 +192,7 @@ type TransactionEvents = 'transaction:start' | 'transaction:end'
 type InstrumentationTypes =
   | 'page-load'
   | 'eventtarget'
+  | 'click'
```

Comment:
add missing type.

Related to: https://github.com/elastic/apm-agent-rum-js/pull/1352#event-9772322079